app/services/parsers/text.rb

```diff
@@ -1,14 +1,30 @@
 module Parsers
-  # Plain text chunker that uses recursive text splitter
+  # Plain text parser
   class Text
-    include RecursiveTextChunker
+    attr_reader :document
 
     def initialize(document)
       @document = document
     end
 
+    # Enumerable chunk records
+    def chunks
+      chonker.chunk(text, text_type)
+    end
+
+    # Override to return the document's text if any additional processing is needed
     def text
       @text ||= @document.contents
     end
+
+    private
+
+    def text_type
+      Chunkers::InputType::PLAIN_TEXT
+    end
+
+    def chonker
```

Comment:
I'm sorry - I was kidding with the suggestion of `chonker`. Feel free to change it back. :D

Reply:
No worries; it's a single private method. No harm.